ui/pages/Applications.tsx

```diff
@@ -16,24 +18,41 @@ type Props = {
 
 function Applications({ className }: Props) {
   const [applications, setApplications] = React.useState<Application[]>([]);
-  const { listApplications, loading } = useApplications();
+  const { applicationsClient, doAsyncError } = React.useContext(AppContext);
+  const [loading, setLoading] = React.useState(false);
+
+  const getApps = () => {
+    setLoading(true);
+    applicationsClient
+      .ListApplications({ namespace: "wego-system" })
+      .then((res) => setApplications(res.applications))
+      .catch((err) => doAsyncError(err.message, err.detail))
+      .finally(() => setLoading(false));
+  };
 
   React.useEffect(() => {
-    listApplications().then((res) => setApplications(res as Application[]));
+    getApps();
+    const interval = setInterval(() => {
+      getApps();
+    }, 30000);
+    return () => clearInterval(interval);
   }, []);
 
   return (
-    <Page loading={loading} className={className}>
-      <Flex align between>
-        <TitleBar>
+    <Page className={className}>
+      <Flex between align>
+        <Flex align>
           <h2>Applications</h2>
-        </TitleBar>
+          <Spacer padding="small" />
+          <PollingIndicator loading={loading} interval="30" />
```

Comment:
This looks a bit misaligned I think, there's a margin-bottom in the css that might be the culprit?